ci/unreal-engine.version

```diff
@@ -1,3 +1,3 @@
 HEAD 4.26-SpatialOSUnrealGDK
 HEAD 4.25-SpatialOSUnrealGDK
-HEAD 4.24-SpatialOSUnrealGDK
+HEAD 4.24-SpatialOSUnrealGDK
```

Comment:
This change okay? I know our CI gets fussy about EOF lines etc. cc: @mironec

Reply:
Should be okay either way in this file, but I guess if it does not have to change, don't change it.